.circleci/config.yml

```diff
@@ -26,7 +26,7 @@ commands:
           # This is necessary because some packages have platform-dependent dependencies
           # For example sqlalchemy requires greenlet only on certain platforms
           name: Freeze requirements
-          command: pip-compile --allow-unsafe ~/project/requirements/requirements.in > ~/project/requirements/requirements.txt
+          command: pip-compile --resolver=backtracking --allow-unsafe ~/project/requirements/requirements.in > ~/project/requirements/requirements.txt
```

Comment:
Can you add this arg to our [bazel targets](https://github.com/sematic-ai/sematic/blob/main/requirements/BUILD#L14) for compiling requirements and re-compile the bazel reqs?

Reply:
Done.